front/src/app/components/admin/conventions/ConventionManageActions.tsx

```diff
@@ -357,6 +358,24 @@ export const ConventionManageActions = ({
           gap: "1rem",
         }}
       >
+        {
+          <Button
+            id={domElementIds.manageConvention.duplicateConventionButton}
+            priority="secondary"
+            linkProps={
+              convention.internshipKind === "immersion"
+                ? routes.conventionImmersion({
+                    ...conventionReadToConventionRouteParams(convention),
+                    skipIntro: true,
+                  }).link
+                : routes.conventionMiniStage(
+                    conventionReadToConventionRouteParams(convention),
+                  ).link
+            }
```

Comment:
on peut simplifier :


```suggestion
            linkProps={routes.conventionImmersion({
                    ...conventionReadToConventionRouteParams(convention),
                    (convention.internshipKind === "immersion" ? { skipIntro: true} : {}),
                  }).link
            }
```